subiquity/server/controllers/install.py

```diff
@@ -151,14 +155,16 @@ async def install(self, *, context):
 
             self.app.update_state(ApplicationState.RUNNING)
 
-            for_install_path = await self.configure_apt(context=context)
+            self.for_install_path = await self.configure_apt(context=context)
```

Comment:
any reason to store this on self?

Reply:
Not anymore, thanks! It was a residual change from before the `AptConfigurer.overlay` context manager.
I'll remove it.